bpf/flows.c

```diff
@@ -178,6 +194,7 @@ static inline int flow_monitor(struct __sk_buff *skb, u8 direction) {
     if (enable_dns_tracking) {
         dns_errno = track_dns_packet(skb, &pkt);
     }
+    track_tls_version(skb, &pkt);
```

Comment:
don't think its better if we have feature config for tracker like we do with all other features ? instead of enabling it by default  ?